spec/features/activity_page_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-describe 'activity page', type: :feature, js: true do
+describe 'activity page', type: :feature, js: true, feature: true do
```

Comment:
Was there not a way to do this besides the `feature: true` hash? That seems redundant after `type: :feature`. Is there not a way to hook into the `type`?